src/vaadin-combo-box-mixin.html

```diff
@@ -623,10 +624,21 @@
       }
     }
 
+    _itemLabelPathChanged(itemLabelPath, oldItemLabelPath) {
+      if (typeof itemLabelPath !== 'string') {
+        console.error('You should set itemLabelPath to a valid string');
+      }
+
+      if (this.selectedItem) {
+        this._inputElementValue = this._getItemLabel(this.selectedItem);
```

Comment:
This logic also already exists [here](https://github.com/vaadin/vaadin-combo-box/blob/master/src/vaadin-combo-box-mixin.html#L692) and it has some additional conditions to it also. Would be cleaner to reuse that logic instead of rewriting something similar here. So the fix would be to change the observer params to `'_selectedItemChanged(selectedItem, itemLabelPath)'`

I also noticed that the following code in the test suite produces different results. Can you check it's not a regression from this PR:

```
combobox.value = null;
combobox.itemLabelPath = 'custom';

console.log(combobox.inputElement.value); // "empty-string-custom"
```

```
combobox.itemLabelPath = 'custom';
combobox.value = null;

console.log(combobox.inputElement.value); // ""
```

Reply:
I have confirmed that this behaviour was there before my changes. If you change the `itemLabelPath` after the `value` change then because the `value` is set to empty string [here](https://github.com/vaadin/vaadin-combo-box/blob/adfe7b783dab90079f92a9ec283cc2427badf498/src/vaadin-combo-box-mixin.html#L679), the input element changes to the empty string value in the items, but if you change the `value` to null after the `itemLabelPath` change then the input element value is set to the empty string value [here](https://github.com/vaadin/vaadin-combo-box/blob/adfe7b783dab90079f92a9ec283cc2427badf498/src/vaadin-combo-box-mixin.html#L683).

Reply:
The `_selectedItemChanged(selectedItem, itemLabelPath)` change is done.